integrated_channels/degreed2/client.py

```diff
@@ -275,6 +277,36 @@ def create_content_metadata(self, serialized_data):
                 f'Degreed2APIClient create_content_metadata failed with status {status_code}: {response_body}',
                 status_code=status_code
             )
+        # once course is created successfully, we need to do 2 more steps
+        # 1. Fetch skills from enterprise-catalog
+        client = EnterpriseCatalogApiClient()
```

Comment:
wouldn't it be better to keep move client in the Degreed2 initialization? Otherwise it will create a new client for every course create and update call.